plugins/woocommerce/client/admin/client/core-profiler/index.tsx

```diff
@@ -615,15 +617,15 @@ const preFetchJetpackAuthUrl = assign( {
 } );
 
 const preFetchGetPlugins = fromPromise( async () =>
-	resolveSelect( ONBOARDING_STORE_NAME ).getFreeExtensions()
+	resolveSelect( onboardingStore ).getFreeExtensions()
 );
 
 const getPlugins = fromPromise( async () => {
-	dispatch( ONBOARDING_STORE_NAME ).invalidateResolution(
+	dispatch( onboardingStore ).invalidateResolutionForStoreSelector(
 		'getFreeExtensions'
 	);
 	const extensionsBundles: ExtensionList[] = await resolveSelect(
```

Comment:
```suggestion
	const extensionsBundles = await resolveSelect(
```
Should we remove these types now? given they come correctly from `getFreeExtensions()` function?
Btw in this instance given they should be identical I don't mind leaving it in either.

Reply:
lgtm, thanks ☺️